server_config.cc

```diff
@@ -0,0 +1,45 @@
+#include <iostream>
+#include "server_config.h"
+#include <string>
+
+ServerConfig::ServerConfig(NginxConfig config) {
+	fillOutMap(config);
+}
+
+ServerConfig::~ServerConfig() {
+  // TODO
+}
+
+void ServerConfig::fillOutMap(NginxConfig config) {
+	// Initialize variables property, value
+	std::string property = "";
+	std::string value = "";
+	for (size_t i = 0; i < config.statements_.size(); i++) {
+		//search in child block
+		if (config.statements_[i]->child_block_ != nullptr) {
```

Comment:
It looks like an editor messed up indentation for all of the blocks that were inside of this for-loop.